data/profiles/spaceweather/WSA.profile

```diff
@@ -0,0 +1,118 @@
+{
+  "assets": [
+    "base",
+    "base_keybindings",
+    "scene/solarsystem/heliosphere/todayssun/actions",
+    "scene/solarsystem/heliosphere/todayssun/mission",
+    "scene/solarsystem/heliosphere/todayssun/whiteBackground",
+    "scene/solarsystem/heliosphere/todayssun/wsa54",
+    "scene/solarsystem/heliosphere/todayssun/grid",
+    "scene/solarsystem/heliosphere/todayssun/sunEarthLine",
+    "scene/solarsystem/planets/earth/earth",
+    "scene/solarsystem/planets/earth/satellites/satellites"
+  ],
+  "camera": {
+    "altitude": 4578000000.0,
+    "anchor": "Sun",
+    "latitude": 1.5877,
+    "longitude": -150.1924,
+    "type": "goToGeo"
+  },
+  "delta_times": [
+    1.0,
+    5.0,
+    30.0,
+    60.0,
+    300.0,
+    1800.0,
+    3600.0,
+    43200.0,
+    86400.0,
+    604800.0,
+    1209600.0,
+    2592000.0,
+    5184000.0,
+    7776000.0,
+    15552000.0,
+    31536000.0,
+    63072000.0,
+    157680000.0,
+    315360000.0,
+    630720000.0
+  ],
+  "keybindings": [
+    {
+      "action": "os.solarsystem.ToggleSatelliteTrails",
+      "key": "S"
+    },
+    {
+      "action": "os.solarsystem.FocusIss",
+      "key": "I"
+    },
+    {
+      "action": "os.solarsystem.FocusEarth",
+      "key": "HOME"
+    }
+  ],
+  "mark_nodes": [
+    "Sun",
+    "Earth",
+    "Mars",
+    "Moon",
+    "Sun",
+    "Venus",
+    "ISS"
+  ],
+  "meta": {
+    "author": "OpenSpace Team",
+    "description": "Default OpenSpace Profile. Adds Earth satellites not contained in other profiles",
```

Comment:
Please make sure to update the meta and in particular this description. It will now show in the user interface

Reply:
Thanks!
How about "This profile uses simulation outputs in real time from the model WSA to visualize the magnetic fluctuations on the Sun." ?